FlutterMockzilla/mockzilla/example/lib/engine/config/mockzilla_config.dart

```diff
@@ -15,7 +15,9 @@ final mockzillaConfig = const MockzillaConfig().addEndpoint(
 
 final defaultResponse = MockzillaHttpResponse(
   statusCode: 200,
-  headers: {"Content-type": "application/json"},
+  // Default headers are {"Content-type": "application/json"},
+  // or you can specify your own
+  headers: {"Content-type": "text/html"},
```

Comment:
I see what you're doing here but I think perhaps leaving it as the [new] default might be better so that this example app still functions as intended?

Reply:
Ah yes true!